tests/limits/test_vif_limit.py

```diff
@@ -12,9 +12,8 @@
 # - a VM (--vm)
 # - the first network on the host can be used to reach the host
 
-vif_limit = 16
-interface_name = "enX"
-vcpus = '8'
+VIF_LIMIT = 16
+VCPUS = '8'
```

Comment:
let's not forget before merging to squash the contents of the "fixup" and "review comments" commits into the relevant ones